lib/streamlit/elements/doc_string.py

```diff
@@ -234,28 +231,12 @@ def _get_variable_name():
 
     The name is "foo"
     """
-    scriptrunner_frame = _get_scriptrunner_frame()
+    code = _get_current_line_of_code_as_str()
+    return _get_variable_name_from_code_str(code)
 
-    if scriptrunner_frame is None:
-        # If there's no ScriptRunner frame, something weird is going on. This
-        # can happen when the script is executed with `python myscript.py`.
-        # Either way, let's bail out nicely just in case there's some valid
-        # edge case where this is OK.
-        return None
 
-    code_context = scriptrunner_frame.code_context
-
-    if not code_context:
-        # Sometimes a frame has no code_context. This can happen inside certain exec() calls, for
-        # example. If this happens, we can't determine the variable name. Just return.
-        # For the background on why exec() doesn't produce code_context, see
-        # https://stackoverflow.com/a/12072941
-        return None
```

Comment:
Ah, I'm guessing this is the small refactor that broke things when running a script using `python3 <script_name>` instead of `streamlit run <script_name>` while keeping the behavior the same when running within streamlit